<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslateScenarioDialog</class>
 <widget class="QDialog" name="TranslateScenarioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>360</height>
   </rect>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="mLatitudeGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="mLatitudeLineEditStatusLabel">
        <property name="toolTip">
         <string>This allows the user to see the reason for a color change of the line edit text.</string>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="mLatitudeLineEditLabel">
        <property name="text">
         <string>Latitude:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="mLatitudeSliderLabel">
        <property name="text">
         <string>Degree:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QSlider" name="mLatitudeSlider">
        <property name="toolTip">
         <string>This allows the user to input the degree of latitude to which to translate.</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QStackedWidget" name="mLatitudeVerticalLineEdit">
        <widget class="QWidget" name="mLatitudeVerticalLineEditDefaultPage"/>
        <widget class="QWidget" name="mLatitudeLineEditPage">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="UtQtLatPosLineEdit" name="mLatitudeLineEdit">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>This allows the user to type in the latitude to which to translate.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="mVerticalLineEditPage">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="UtQtLengthValueEdit" name="mVerticalLineEdit">
            <property name="toolTip">
             <string>This allows the user to type in the vertical distance by which to translate.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mLongitudeGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="mLongitudeLineEditStatusLabel">
        <property name="toolTip">
         <string>This allows the user to see the reason for a color change of the line edit text.</string>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="mLongitudeSliderLabel">
        <property name="text">
         <string>Degree:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QSlider" name="mLongitudeSlider">
        <property name="toolTip">
         <string>This allows the user to input the degree of longitude to which to translate.</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QStackedWidget" name="mLongitudeHorizontalLineEdit">
        <widget class="QWidget" name="mLongitudeHorizontalLineEditDefaultPage"/>
        <widget class="QWidget" name="mLongitudeLineEditPage">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="UtQtLonPosLineEdit" name="mLongitudeLineEdit">
            <property name="toolTip">
             <string>This allows the user to type in the longitude to which to translate.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="mHorizontalLineEditPage">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="UtQtLengthValueEdit" name="mHorizontalLineEdit">
            <property name="toolTip">
             <string>This allows the user to type in the horizontal distance by which to translate.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="mLongitudeLineEditLabel">
        <property name="text">
         <string>Longitude:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="MapUtils::ScenarioTransformation::DialogMenuAndButtonsWidget" name="mDialogMenuAndButtons" native="true"/>
   </item>
   <item>
    <spacer name="mSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapUtils::ScenarioTransformation::DialogMenuAndButtonsWidget</class>
   <extends>QWidget</extends>
   <header>DialogMenuAndButtonsWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UtQtLatPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLonPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLengthValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
